### DISCUSSION TITLE
a way to prevent jobs running on a server without having to shutdown Cronicle on that server

### DISCUSSION THREAD

**Discussion by kranskydog** [jhuckaby/Cronicle#669] [Ideas]
We may have to disable all the jobs from running on a particular server for (reasons) but while I can shutdown Cronicle on that server from the GUI, I have to login to the server as root to start Cronicle again.

A way to pause Cronicle on that server, i.e. telling the Master to not schedule any jobs for that server, would be useful.

**Comment by jhuckaby** [comment]
Yup, excellent suggestion.  This is actually already implemented in Cronicle v2.  I'll see if I can somehow backport this to v0 easily (no promises tho).
